govcd/api.go

```diff
@@ -125,8 +127,13 @@ func ContainsNotFound(err error) bool {
 	return err != nil && strings.Contains(err.Error(), ErrorEntityNotFound.Error())
 }
 
-// NewRequestWitNotEncodedParams allows passing complex values params that shouldn't be encoded like for queries. e.g. /query?filter=name=foo
+// NewRequestWitNotEncodedParamsWithApi allows passing complex values params that shouldn't be encoded like for queries. e.g. /query?filter=name=foo
```

Comment:
`NewRequestWitNotEncodedParamsWithApi` doesn't seem to match real function name.

Reply:
updated